aea/contracts/base.py

```diff
@@ -303,7 +303,7 @@ def _load_contract_interfaces(
         full_path = Path(configuration.directory, path)
         if identifier not in ledger_apis_registry.supported_ids:
             raise ValueError(  # pragma: nocover
-                "No ledger api registered for identifier {}."
+                "No ledger api registered for identifier {}.".format(identifier)
```

Comment:
Please use f strings where possible in the future

Reply:
It's what I usually do, but I decided to stick with the style already present in that file. I'll keep that in mind.